contracts/margin/external/BucketLender/BucketLender.sol

```diff
@@ -762,8 +762,7 @@ contract BucketLender is
             lockedBucket = criticalBucket;
         }
 
-        uint256 totalOwedToken = 0;
-        uint256 totalHeldToken = 0;
+        uint256[2] memory results; // [0] = owedToken, [1] = heldToken
```

Comment:
why was this changed?

Reply:
So the modifier variable counts toward the 16-var stack limit of functions >:(

Reply:
make this `// [0] = totalOwedToken, [1] = totalHeldToken`